checks.d/network.py

```diff
@@ -508,3 +554,71 @@ def _parse_solaris_netstat(self, netstat_output):
             metrics_by_interface[iface] = metrics
 
         return metrics_by_interface
+
+    def _linux_sockstat_get_conntrack_max(self):
+        proc_name = '/proc/sys/net/ipv4/ip_conntrack_max'
+        with opened_w_error(proc_name,'r') as (proc, err):
+            if err:
+                self.log.info("Unable to read %s - %s" % (proc_name, str(err)))
+            else:
+                return int(proc.readline())
+
+    def _linux_sockstat_get_conntrack_count(self):
+        proc_name = '/proc/sys/net/ipv4/netfilter/ip_conntrack_count'
+        with opened_w_error(proc_name,'r') as (proc, err):
+            if err:
+                self.log.info("Unable to read %s - %s" % (proc_name, str(err)))
+            else:
+                return int(proc.readline())
+
+    def _linux_sockstat_get_max_orphans(self):
+        proc_name = '/proc/sys/net/ipv4/tcp_max_orphans'
+        with opened_w_error(proc_name,'r') as (proc, err):
+            if err:
+                self.log.info("Unable to read %s - %s" % (proc_name, str(err)))
+            else:
+                return int(proc.readline())
+
+    def _linux_sockstat_get_size_of_page(self):
+        return long(check_output(["/usr/bin/getconf", "PAGESIZE"]))
+
+    def _linux_sockstat_get_max_memory(self):
```

Comment:
Much of this function is redundant as well, except for the else statement. Maybe this could be condensed in another way. I'm looking to remove repeated code in here, whenever possible.